File: BOJ/2839.cpp
#include <iostream>

int main() {
    int n, cnt = 0;
    scanf("%d", &n);
    for (;n > 0 && n % 5; n -= 3, cnt++);
    printf("%d", n < 0 ? -1 : n / 5 + cnt);
    return 0;
}

File: BOJ/2839.js
let sc = Number(require('fs').readFileSync('/dev/stdin').toString());
let cnt = 0;
for (;sc % 5; sc -= 3) {
    if (sc < 0) break;
    cnt += 1;
}
console.log(sc < 0 ? -1 : cnt + sc / 5);